pkg/eventprocessor/impl/executor/impl/txnscope_runsql.go

```diff
@@ -0,0 +1,524 @@
+package impl
```

Comment:
Notice how separating logic per event type in files shows the complexity of the event type execution implementation.